### PR TITLE
[KOGITO-914] - Create rpm package for Kogito CLI

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.redhat.com/browse/KOGITO-914

Signed-off-by: Mauricio Magnani <msmagnanijr@gmail.com>

### CODE REVIEW

**Comment by msmagnanijr** [comment]
I closed the other pull request wrongly,  please check this one.  @ricardozanini I did the changes according to your recommendations.

File: kogito-cli-rpm-Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+pipeline{
+    agent {label 'kogito-cloud && !master'}
+    parameters {
+        string(name: 'KOGIT0_CLI_BINARY_URL', defaultValue: '', description: 'Kogito binary .tar.gz file')
```
Comment (radtriste) [inline kogito-cli-rpm-Jenkinsfile:4]: Is there a way to construct the `KOGIT0_CLI_BINARY_URL` with the `KOGITO_CLI_VERSION` ?

File: kogito-cli-rpm-Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        string(name: 'KOGITO_CLI_VERSION', defaultValue: '', description: 'Kogito CLI version')
+        string(name: 'KOGITO_CLI_RPM_CHANGELOG', defaultValue: '', description: 'Changelog must be in this format Thu Jul 02 2020  Your Name <yourname@redhat.com>')
```
Comment (radtriste) [inline kogito-cli-rpm-Jenkinsfile:6]: Can't we get that info from Github release page ?

File: kogito-cli-rpm-Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        string(name: 'KOGITO_CLI_RPM_CHANGELOG_DESCRIPTION', defaultValue: '', description: 'Changelog description')
```
Comment (radtriste) [inline kogito-cli-rpm-Jenkinsfile:7]: also for that one

File: tools/create-kogito-cli-rpm.sh [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+mkdir -p ~/rpmbuild/{RPMS,SRPMS,BUILD,SOURCES,SPECS,tmp}
+
+cat <<EOF >~/.rpmmacros
+%_topdir   %(echo $HOME)/rpmbuild
+%_tmppath  %{_topdir}/tmp
+EOF
+
+cd ~/rpmbuild
+wget ${KOGIT0_CLI_BINARY_URL}
```
Comment (radtriste) [inline tools/create-kogito-cli-rpm.sh:11]: Where is that var coming from ?
You are calling the script with arguments but never retrieve them here.
Either you use env variables or you use parameters to the script ;)

**Comment by spolti** [comment]
cool!.

we cannot merge this yet, this issue will be discussed for next releases.

**Comment by msmagnanijr** [comment]
> cool!.
> 
> we cannot merge this yet, this issue will be discussed for next releases.

Sure! In the meantime if anyone wants to try RPM it is here:

sudo dnf install dnf-plugins-core
dnf copr enable msmagnanijr/kogito-cli
sudo dnf install kogito-cli

https://copr.fedorainfracloud.org/coprs/msmagnanijr/kogito-cli

File: kogito-cli-rpm-Jenkinsfile (hunk 1 above)
Comment (ricardozanini) [reply kogito-cli-rpm-Jenkinsfile:4]: That's what I mentioned in the last PR.

File: kogito-cli-rpm-Jenkinsfile (hunk 2 above)
Comment (ricardozanini) [reply kogito-cli-rpm-Jenkinsfile:6]: Yes!

**Pull request closed without merging**